SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# Security Policy
+
+## Supported Versions
+
+Security vulnerability reports will be accepted and acted upon for all released versions.
```

Comment:
Maybe a bit too broad and something like this would be better?

https://github.com/mne-tools/mne-python/blob/main/SECURITY.md#supported-versions